src/wrap-store/shallowDiff.js

```diff
@@ -0,0 +1,36 @@
+const STATUS_UPDATED = 'updated';
```

Comment:
Love this direction. Let's put these in a shared `src/constants` file and reference them here and in `Store.js` above.